### PR TITLE
fix siconca variable in EC-Earth3 and EC-Earth3-Veg models in amip simulation

### LINKED ISSUE #703: Issue with siconca variable in EC-Earth-Consortium models in CMIP6 amip simulation
**Describe the bug**
I assume that there is an issue with **siconca** variable for both models from EC-Earth-Consortium, i.e. **EC-Earth3** and **EC-Earth3-Veg**  in the amip simulation. Sea-ice area should be provided in % and vary from 0 to 100.  In the description of the .nc files from these two models despite it is indicated that units are "%" and comment "Percentage of grid cell covered by sea ice", the values vary from 0 to 1, which stands for the fraction. 

Of course, this affects multi-model mean calculation within ESMValTool.

**Please attach**

Please, see for example: 
/work/ik1017/CMIP6/data/CMIP6/CMIP/EC-Earth-Consortium/EC-Earth3/amip/r1i1p1f1/SImon/siconca/gr/v20200203/
_e.g._: `ncdump -v siconca siconca_SImon_EC-Earth3_amip_r1i1p1f1_gr_197901-197912.nc`

### PR DESCRIPTION
**Pull request by egalytska**
Before you start, please read our [contribution guidelines](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html).

**Tasks**

-   [x] [Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do, if you haven't done so already (and add the link at the bottom)
-   [x] This pull request has a descriptive title that can be used in a changelog
-   [x] Add unit tests
-   [x] Public functions should have a numpy-style docstring so they appear properly in the [API documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/api/esmvalcore.html). For all other functions a one line docstring is sufficient.
-   [x] Circle/CI tests pass. Status can be seen below your pull request. If the tests are failing, click the link to find out why.
-   [ ] Codacy code quality checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why. If you suspect Codacy may be wrong, please ask by commenting.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #703

### CODE REVIEW

File: esmvalcore/cmor/_fixes/cmip6/ec_earth3.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        cube: iris.cube.Cube
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        metadata = cube.metadata
+        cube *= 100
+        cube.metadata = metadata
```
Comment (bouweandela) [inline esmvalcore/cmor/_fixes/cmip6/ec_earth3.py:25]:
```suggestion
        cube.data = cube.core_data() * 100.
```
would be recommended, as that surely only changes the required data.

**Comment by egalytska** [comment]
@bouweandela could you please have a look at new changes that include your suggestion. 
Btw, Codacy is still complaining =/

**Comment by valeriupredoi** [comment]
@egalytska - done and I added two tests for your fixes. I think this is ready to go :+1:

**Review by valeriupredoi — APPROVED** [review]

**Comment by egalytska** [comment]
> @egalytska - done and I added two tests for your fixes. I think this is ready to go

@valeriupredoi Thanks a lot!!

**Pull request merged**